instrumentation/armeria/armeria-1.3/library/src/main/java/io/opentelemetry/instrumentation/armeria/v1_3/ArmeriaTelemetryBuilder.java

```diff
@@ -102,7 +63,8 @@ public ArmeriaTelemetryBuilder setStatusExtractor(
   @CanIgnoreReturnValue
   public ArmeriaTelemetryBuilder addAttributeExtractor(
       AttributesExtractor<? super RequestContext, ? super RequestLog> attributesExtractor) {
-    additionalExtractors.add(attributesExtractor);
+    clientBuilder.addAttributeExtractor(attributesExtractor);
+    serverBuilder.addAttributesExtractor(attributesExtractor);
```

Comment:
probably should rename this method (separate PR would be easiest to review, I think we still have some decisions to make in this PR)
```suggestion
    clientBuilder.addAttributesExtractor(attributesExtractor);
    serverBuilder.addAttributesExtractor(attributesExtractor);
```